src/app/wallet/wallet/shared/signature-address-modal/signature-address-modal.component.ts

```diff
@@ -70,7 +71,7 @@ export class SignatureAddressModalComponent implements OnInit {
   openLookup(): void {
     const dialogRef = this.dialog.open(AddressLookupComponent);
     // @TODO confirm lookup type
-    dialogRef.componentInstance.type = (this.type === 'sign') ? 'receive' : 'send';
+    dialogRef.componentInstance.type = (this.type === 'sign') ? 'sign' : 'send';
```

Comment:
Huh, 'send'?

Reply:
Yes `send` because we only verify address book address